<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemWordWdg</class>
 <widget class="QWidget" name="RemWordWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>背单词</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnWordList">
        <property name="text">
         <string>单词表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWordRember">
        <property name="text">
         <string>默写</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTool">
        <property name="text">
         <string>工具</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWordExample">
        <property name="text">
         <string>样例</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="funWdg" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
